lib/signalfx/tracing.rb

```diff
@@ -44,6 +49,17 @@ def configure(tracer: nil,
           Compat.apply
         end
 
+        def process_span_tags(span_tags: {})
+          span_tags_string = ENV['SIGNALFX_SPAN_TAGS']
+          if not span_tags_string.nil?
+            span_tags_string.split(',').each do |t|
```

Comment:
fyi SFx.NET current implementation is comma-delimited for the key-value pairs and the pairs themselves are separated by a colon.

Reply:
Updated.